.github/workflows/ci.yml

```diff
@@ -315,6 +315,46 @@ jobs:
           flags: ${{ steps.test.outputs.codecov_flag }}
           fail_ci_if_error: false
 
+  macos-test:
+    needs: prep-testbed
+    runs-on: macos-latest
+    env:
+      JINA_DAEMON_BUILD: DEVEL
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Prepare enviroment
+        run: |
+          brew install docker-machine docker
+          docker-machine create default
+          docker-machine env default
+          eval $(docker-machine env default)
+          brew services start docker-machine
+          docker build -f Dockerfiles/pip.Dockerfile -t jinaai/jina:test-pip .
+          python -m pip install --upgrade pip
+          python -m pip install wheel
+          pip install ".[standard,test,cicd]" --no-cache-dir
+          jina
+          export JINA_LOG_LEVEL="ERROR"
```

Comment:
Because each step in github actions is run in its own process, this will not persist. Set it when you do the test, or globally for the whole job